src/main/java/uk/gov/digital/ho/hocs/search/api/CaseDataService.java

```diff
@@ -56,12 +57,16 @@ public class CaseDataService {
 
     private final CaseTypeMappingRepository caseTypeMappingRepository;
 
+    private final FieldQueryTypeMappingRepository fieldQueryTypeMappingRepository;
```

Comment:
I wonder if we should inject this into the HocsQueryBuilder class? It doesn't appear to be used within the case data service. Or alternatively get the query type from the repository within this method and pass the search type through to the query builder?